composer.json

```diff
@@ -1,12 +1,51 @@
 {
     "name": "zendframework/zend-coding-standard",
-    "description": "Zend Framework coding standard",
+    "type": "phpcodesniffer-standard",
```

Comment:
I'm so happy that composer has made things like plugin registration automated via package `type`!